### ISSUE TITLE
typescript builder path/file

### ISSUE THREAD

**Issue by pinoyyid** [c9/core#26]
Using the TS builder, the generated JS for a file 'foo.ts' is 'foo/.js' whereas one would expect it to be 'foo.js'

Not sure if this is a bug. If it is , please remove the slash. If not, please can I get a pointer to the default TS builder so I can use it to create my own.

**Comment by javruben** [comment]
This must be a bug. You can find the builder here:
https://github.com/c9/c9.ide.run.build/blob/master/builders/typescript.build

We'd appreciate a pull request with a fix. I think the / in that regexp should be removed.

**Comment by pinoyyid** [comment]
Thanks for confirming and the link. I'd love to offer a PR, but I don't know enough about the sublime/c9 build mechanism to do so with any confidence.

My local ts build, which I've implemented and seems to behave correctly is

    {
        "cmd": ["tsc",  "$file"],
        "selector": "source.ts"
    }

as opposed to 
    {
        "cmd": ["tsc", "--out", "${file/\\.ts/\\.js/}", "$file"],
        "selector": "source.ts"
    }

ie. I've removed the --out argument altogether, which causes tsc to compile to $file.js which is the behaviour I'm looking for. I'm reluctant to propose that as a PR without knowing if that approach breaks elsewhere.
